src/sensors.c

```diff
@@ -365,6 +365,7 @@ static int sensors_load_conf(void) {
       if ((feature->type != SENSORS_FEATURE_IN) &&
           (feature->type != SENSORS_FEATURE_FAN) &&
           (feature->type != SENSORS_FEATURE_TEMP) &&
+          (feature->type != SENSORS_FEATURE_HUMIDITY) &&
```

Comment:
Can you please move your added code under #if SENSORS_API_VERSION >= 0x402 condition?
Thanks.

Reply:
This is fixed now!